src/components/GuardianCard.tsx

```diff
@@ -0,0 +1,61 @@
+import { Gray1, Gray3, Gray4, TextDark, TextLight } from '../colors';
+
+import React from 'react';
+import { rem } from 'polished';
+import styled from 'styled-components';
+
+type Props = {
+  address: string;
+  status: number;
+};
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  /* justify-content: center; */
+  align-items: center;
+  border: 1px solid ${Gray3};
+  width: ${rem(240)};
+  height: ${rem(230)};
```

Comment:
discrete height & width work 100% for now. once this is integrated into the page though, there's a chance we can make the width responsive via the `flex` shorthand property

Reply:
Are you saying that once we have them on a page we should use flex maybe, and the boxes will grow with the window size?